src/search/include/search/search_tree.h

```diff
@@ -21,7 +21,7 @@
 
 #include "automata/ta_regions.h"
 #include "canonical_word.h"
-#include "preorder_traversal.h"
+//#include "preorder_traversal.h"
```

Comment:
We do not need it? If not, let us remove the commented include

Reply:
Yes so unfortunately, the tree traversal completely breaks in a search graph, as it is no longer a tree and thus we cannot traverse it easily anymore.

We should probably fix this, for now I have just removed the tree traversal.